nautobot/extras/tests/test_filters.py

```diff
@@ -50,12 +59,11 @@
     Tag,
     Webhook,
 )
-from nautobot.ipam.models import IPAddress, VLAN
+from nautobot.ipam.models import VLAN, IPAddress
```

Comment:
Why this change?

Reply:
Maybe I'm using a different Isort profile?  I have isort running on with the `black` profile. I've tried the `django` profile as well and it does even more.  I don't see anything in the styling guide for import sorting.  I can manually change it back if required.

Reply:
from the isort docs as well.
```
NOTE: type here refers to the implied type from the import name capitalization. isort does not do type introspection for the imports. These "types" are simply: CONSTANT_VARIABLE, CamelCaseClass, variable_or_function
```

Reply:
Yeah, we don't have isort as part of the standard linters for this project at this time. Let's leave the import order alone for now to reduce churn where we're not otherwise making changes.

Reply:
pushed the fix Had to add "--dont-order-by-type" to my isort